Zend/zend_execute.c

```diff
@@ -3274,6 +3274,15 @@ ZEND_API void zend_free_compiled_variables(zend_execute_data *execute_data) /* {
  *                             +----------------------------------------+
  */
 
+/* zend_copy_extra_args is used when the actual passed number of arguments
+ * (EX_NUM_ARGS) is greater than what the function defined (op_array->num_args)
+ *
+ * the extra arguments will be copied into the call frame after all the compiled variables.
```

Comment:
nit: Please capitalize the first letter of each sentence , at least for large comments like these.

Reply:
Thanks, fixed.